earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -24,10 +24,19 @@ sudo yum install -y epel-release
 
 ### RHEL 6
 ```
+# For RHEL 6 Workstation:
 sudo subscription-manager repos --enable=rhel-x86_64-workstation-dts-2
+
+# For RHEL 6 Server:
+sudo subscription-manager repos --enable=rhel-server-dts2-rpms
+
+# For all RHEL 6 Editions:
 sudo subscription-manager repos --enable=rhel-6-server-optional-rpms
+sudo yum install https://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
May want to change this link to `https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm`

Reply:
That assumes they have the latest installed. I suppose. I'll need to change the note to say something about needing to match if they don't have the latest RHEL

Reply:
I have 6.9 installed but there's no epel-release-6-9.noarch.rpm.  I think the 6.8 works, and I know the latest works, so that's why I suggested the change.

Reply:
Oh, I made this comment in another file, but I use
```
https://dl.fedoraproject.org/pub/epel/epel-release-latest-${RED_HAT_VERSION_MAJOR}.noarch.rpm
```

It doesn't specify a particular RHEL version, or processor architecture.